configuration/src/secrets.rs

```diff
@@ -160,15 +185,16 @@ mod test {
                 let secrets =
                     AgentSecrets::from_env(networks).expect("Failed to load secrets from env");
 
-                let default_config =
-                    TxSubmitterConf::Ethereum(ethereum::TxSubmitterConf::Local(SignerConf::Aws {
-                        id: "default_id".into(),
-                    }));
+                let default_config_ethereum = ethereum::TxSubmitterConf::Local(SignerConf::Aws {
+                    id: "default_id".into(),
+                });
                 for config in secrets.tx_submitters.values() {
-                    assert_eq!(*config, default_config);
-                }
-                for config in secrets.rpcs.values() {
-                    assert!(matches!(*config, ChainConf::Ethereum { .. }));
+                    match config {
+                        TxSubmitterConf::Ethereum(conf) => {
+                            assert_eq!(*conf, default_config_ethereum)
+                        }
+                        TxSubmitterConf::Substrate(_) => continue,
```

Comment:
Clarification: Why are we using a `continue` here

Reply:
Not really interested in checking substrate value as this is really a check to ensure ethereum default override worked